Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -2645,12 +2645,15 @@ internal void ApplySettings()
                 runPlayerUpdatesInEditMode = m_Settings.IsFeatureEnabled(InputFeatureNames.kRunPlayerUpdatesInEditMode);
                 #endif
 
+                // Ignore
```

Comment:
I would suggest to remove the comment or change it to say something like "Prevent error messages - don't run when WGI feature flag is explicitly set on non-Windows platforms"